src/angular/src/app/pages/files/file-options.component.html

```
@@ -142,6 +142,18 @@
                         <span class="text">Name Z&#8594;A</span>
                     </div>
                 }
+                @if ((options | async)?.sortMethod === SortMethod.SIZE_ASC) {
+                    <div class="sel-item">
+                        <div class="icon"><img src="assets/icons/sort-asc.svg" class="sort-asc" /></div>
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)